### PR TITLE
Automatically rerun failed scenarios

### PR DESCRIPTION
**Pull request by wojsmol**
fixes https://github.com/wp-cli/wp-cli/issues/4901

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,15 +14,16 @@
         "rmccue/requests": "~1.6",
         "symfony/finder": ">2.7",
         "wp-cli/mustangostang-spyc": "^0.6.3",
-        "wp-cli/php-cli-tools": "~0.11.2"
+        "wp-cli/php-cli-tools": "~0.11.2",
+        "wp-cli/wp-cli-tests": "^2.0.7"
```
Comment (schlessera) [inline composer.json]: This is a duplicate and needs to be removed (as it is also in the wrong section).

File: .travis.yml [hunk 2]
```diff
@@ -41,7 +41,7 @@ install:
 
 script:
   - composer phpunit
-  - composer behat
+   - composer behat || composer behat-rerun
```
Comment (schlessera) [inline .travis.yml]: There's an issue with the indentation here.

Comment (wojsmol) [reply .travis.yml]: fixed in https://github.com/wp-cli/wp-cli/pull/4906/commits/4a782e0b8822dcd941c7812fb0956cd0bec2545e

File: composer.json (hunk 1 above)
Comment (wojsmol) [reply composer.json]: fixed in https://github.com/wp-cli/wp-cli/pull/4906/commits/4a782e0b8822dcd941c7812fb0956cd0bec2545e

Comment (schlessera) [reply composer.json]: No, the line is still there. https://github.com/wp-cli/wp-cli/commit/4a782e0b8822dcd941c7812fb0956cd0bec2545e only fixed the indentation problem.

**Pull request merged**